GatherPress/gatherpress
# GP 623: Revert sprintf so logged out rsvp functionality works

### Description of the Change
Removing sprintf so the logged out version of the RSVP block functions properly

Closes #623 

### How to test the Change
Try to RSVP to an event when logged out and ensure the log/register links work properly.

![Screenshot 2024-03-31 at 11 37 37 AM](https://github.com/GatherPress/gatherpress/assets/223919/5ab708ef-983f-4b7d-9cb6-08e208b7f464)


### Changelog Entry

> Fixed - Bug fix


### Checklist:
